### ISSUE TITLE
Entity Framework Core as configuration store

### ISSUE THREAD

**Issue by koryphaee** [aloneguid/config#101]
I am trying to store my config values in a simple key-value table in EF Core. Am I missing something or is this not implemented yet? I also did not find a third-party NuGet package which has such an implementation.

Since I like the concept of Config.Net I would go ahead and implement this feature if it doesn't exists yet. I would then submit a pull request to make this functionality available to anyone.

**Comment by aloneguid** [comment]
You're correct, there is no EF Core provider at the moment. Please go ahead with PR, it will be a great addition.

**Comment by koryphaee** [comment]
Pull request here: https://github.com/aloneguid/config/pull/102

I tried copying your coding style as much as possible to make the code look uniform. If I missed something please go ahead and adjust it. I also tried to include comments where it makes sense and wrote some tests and documentation.

**Comment by feitzi** [comment]
Any news?
